docs/DafnyRef/UserGuide.md

```diff
@@ -365,7 +365,21 @@ The command emits exit codes of
 
 #### 25.5.1.9. `dafny format` {#sec-dafny-format}
 
-This command is not yet released, but will be a command that formats source code to a consistent style.
+Dafny supports a formatter, which for now only changes the indentation of every line in a Dafny file, so that it conforms
+to the idiomatic Dafny code formatting style.
+For the formatter to work, the file should be parsed correctly by Dafny.
+
+There are four ways to use the formatter:
+
+* `dafny format file.dfy [file2.dfy ...]` formats the given Dafny files, altering the files in place.
+* `dafny format folder` formats all the Dafny files contain in the given folders, altering the files in place. For example, `dafny format .` formats all the Dafny files recursively in the current folder.
```

Comment:
Can one have multiple folders? Mix them with files?

Reply:
Yes to both. Do you have a suggestion on how to word it efficiently?